manifests/config.pp

```diff
@@ -53,6 +53,14 @@
     content => "This directory is managed by puppet but *.conf files are ignored from purging\n",
     require => File['/etc/cvmfs/domain.d'],
   }
+  file{'/etc/cvmfs/config.d/README.PUPPET':
```

Comment:
Hi,
The domain.d/README and config.d/README may as well be an array off two files like you did for the
directory.